.travis.yml

```diff
@@ -4,6 +4,6 @@ node_js:
 services:
   - mongodb
 env:
-  - MONGODB_URI=mongodb://127.0.0.1/
+  - MONGODB_URI=mongodb://127.0.0.1/ JWT_SECRET=somesecret
```

Comment:
Yeah so we have this JWT_SECRET variable for signing the JWT's and stuff, Travis needs it for the tests. As it's just going to be on the test database I don't think we need to encrypt this, but we can.